editor/styling/stylesheets/modules/_console.scss

```diff
@@ -233,11 +233,11 @@
   -fx-text-fill: -df-text;
 
   &:focused {
-    -fx-background-color: -df-background-input;
-    -fx-border-color: -df-component-dark;
+    -fx-background-color: -df-background-light;
+    -fx-border-color: -df-background-lighter;
     -fx-border-width: 1px 0px 0px 0px;
     -fx-padding: 0px;
-    -fx-text-fill: -df-text-light;
+    -fx-text-fill: -df-text;
```

Comment:
Unrelated: Fixed a CSS regression visible when focusing on the **Evaluate Lua** field in the debugger.